sphinxlint/utils.py

```diff
@@ -121,17 +121,21 @@ def looks_like_glued(match):
     return True
 
 
+_START_OF_COMMENT_BLOCK_PATTERN = re.compile(r"^\s*\.\.$")
+_PRODUCTION_LIST_DIRECTIVE_PATTERN = re.compile(r"^ *.. productionlist::")
+_COMMENT_PATTERN = re.compile(r"^ *\.\. ")
```

Comment:
This one can also be used in the `hide_non_rst_blocks()` function immediately below `is_multline_non_rst_block()`

Reply:
True! I hadn't realised because I was focusing on the output of the profiling, but you're absolutely right, I'll reuse it there too.